nodejs/week1/meal_app/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
+
+const cheapMealsArray = mealsJson.filter(cheapMeals => cheapMeals.price < 50)
+cheapMealsArray.forEach(meal => {
+    meal.review = [];
```

Comment:
You can write it shorter with filter:

```
meal.review = reviewsJson.filter(mealReview => mealReview.mealId === meal.id)
```

If you have questions regarding how it works, ask me on Sunday ;)

The same code applicable to all the places you used similar logic.